src/input/reader_to_storage.cc

```diff
@@ -632,7 +633,7 @@ StorageBase * ReaderToStorage::make_storage_from_default(const string &dflt_str,
 
```

Comment:
Type string is necessary. Method check_validity creates storage only for Default with value at declaration and and it is thus ensured. For other types of Default method "does nothing" (storage == null) and subsequently causes segfault.